tests/ioctl_v4l2.c

```diff
@@ -1334,6 +1333,27 @@ main(void)
 	       "})" RVAL_EBADF,
 	       XLAT_STR(VIDIOC_QUERY_EXT_CTRL));
 
+	ioctl(-1, VIDIOC_EXPBUF, 0);
+	printf("ioctl(-1, %s, NULL)" RVAL_EBADF,
+	       XLAT_STR(VIDIOC_EXPBUF));
+
+	struct v4l2_exportbuffer *const p_v4l2_expbuf =
+	       page_end - sizeof(*p_v4l2_expbuf);
+	ioctl(-1, VIDIOC_EXPBUF, p_v4l2_expbuf);
+	printf("ioctl(-1, %s, {"
+	       "type=%#x" NRAW(" /* V4L2_BUF_TYPE_??? */") ", "
+	       "index=%u, "
+	       "plane=%u, "
+	       "flags="
+	       NABBR("0xcfcecdcc") VERB(" /* ")
+	       NRAW("O_RDONLY|O_CREAT|O_EXCL|O_NOCTTY|O_APPEND|O_NONBLOCK|O_DIRECT|O_LARGEFILE|O_NOFOLLOW|O_NOATIME|O_CLOEXEC|__O_TMPFILE|0xcf80000c")
```

Comment:
I'm afraid hardcoding this is not portable as values for some of these constants, e.g. O_LARGEFILE, differ between architectures.

Reply:
The test fails with
```diff
-ioctl(-1, VIDIOC_EXPBUF, {type=0xffffffff /* V4L2_BUF_TYPE_??? */, index=123, plane=321, flags=O_CLOEXEC}) = -1 EBADF (Bad file descriptor)
+ioctl(-1, VIDIOC_EXPBUF, {type=0xffffffff /* V4L2_BUF_TYPE_??? */, index=123, plane=321, flags=O_RDONLY|O_CLOEXEC}) = -1 EBADF (Bad file descriptor)
```
Looks like it might be a good idea to do `p_v4l2_expbuf->flags = O_RDONLY|O_CLOEXEC` and `NRAW("O_RDONLY|O_CLOEXEC")`.